### PR TITLE
Create cluttered_mnist_rnn , implementation for recurrent spatial transformer 

### PR DESCRIPTION
**Pull request by seragENTp**
tensorflow implementation for recurrent spatial transformer : https://arxiv.org/abs/1509.05329

### CODE REVIEW

**Comment by seragENTp** [comment]
I signed the CLA .

**Comment by drpngx** [comment]
@ebrevdo is that something you want to review?

@seragENTp Please reformat to our style guide. The extension of the filename should be .py and you should import futures.

**Comment by daviddao** [comment]
@drpngx the commited file needs a .py ending. I will review it.

**Comment by seragENTp** [comment]
@drpngx sorry, but what is your style guide ?

**Comment by drpngx** [comment]
https://www.tensorflow.org/versions/r0.9/how_tos/style_guide.html

**Comment by nealwu** [comment]
@daviddao are you still planning to review?

File: transformer/cluttered_mnist_rnn.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+l_conv2_loc = tf.reshape(l_conv2_loc,[-1 ,9*9*20 ])
+
+# Replicate input for Gated Recurrent Unit
+l_conv2_loc = tf.tile(l_conv2_loc,[1,3])
+
+l_conv2_loc = tf.split(1,3,l_conv2_loc)
+
+# Gated Recurrent Unit
+gru_cell = rnn_cell.GRUCell(num_units=256)
+
```
Comment (daviddao) [inline transformer/cluttered_mnist_rnn.py]: Would you mind removing the empty lines between the model definition? Please follow the style guide.

File: transformer/cluttered_mnist_rnn.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import tensorflow as tf
+from spatial_transformer import transformer
+from tensorflow.python.ops import rnn,rnn_cell
```
Comment (daviddao) [inline transformer/cluttered_mnist_rnn.py]: This doesn't work for tf 1.0 anymore, please also stay consistent with the empty spaces

**Comment by daviddao** [comment]
@seragENTp would you mind to adapt your PR to tensorflow 1.0?

**Comment by nealwu** [comment]
Closing old PRs.

**Pull request closed without merging**